test/e2e/node/pods.go

```diff
@@ -46,13 +46,13 @@ var _ = SIGDescribe("Pods Extended", func() {
 		BeforeEach(func() {
 			podClient = f.PodClient()
 		})
-		// TODO: Fix Flaky issue #68066 and then re-add this back into Conformance Suite
```

Comment:
How do we know this has been fixed?  Like, beyond "that issue was closed", how can I tell that this test is no longer flaky?

Reply:
@spiffxp based on the comment https://github.com/kubernetes/kubernetes/pull/64240#discussion_r191465289 , this test case was marked as `[flaky]` because of watch. The watch is removed now and its okay to promote conformance, because its not a flaky now.

Reply:
>because its not a flaky now.
what evidence or data can you provide to back up this claim?  all you've told me is that some code changed

Reply:
https://github.com/kubernetes/kubernetes/pull/75473#issuecomment-479857894